packages/react-query/src/prices/usePrices.tsx

```diff
@@ -0,0 +1,105 @@
+import { useQuery } from '@tanstack/react-query';
+import { useCallback } from 'react';
+
+enum PriceCurrency {
+  USD = 'usd',
+  BTC = 'btc',
+  EUR = 'eur'
+}
+
+enum CurrencyTickers {
+  ALEX = 'ALEX',
+  DAI = 'DAI',
+  DLLR = 'DLLR',
+  ESOV = 'eSOV',
+  ETH = 'ETH',
+  RETH = 'rETH',
+  STONE = 'STONE',
+  TBTC = 'tBTC',
+  USDC = 'USDC',
+  USDT = 'USDT',
+  WBTC = 'WBTC',
+  WSTETH = 'wstETH',
+  BTC = 'BTC'
+}
+
+// TODO: Add other supported currencies. Move.
+// UPDATE: use assets file where each asset has an apiId
+const COINGECKO_IDS = [
+  'alexgo',
+  'dai',
+  'ethereum',
+  'sovryn',
+  'rocket-pool-eth',
+  'stakestone-ether',
+  'tbtc',
+  'tether',
+  'usd-coin',
+  'wrapped-bitcoin',
+  'wrapped-steth',
+  'bitcoin',
+  'sovryn-dollar'
+];
```

Comment:
Ideally these live elsewhere but this is fine for now

Reply:
I didn't fully get to clean up the react-query package. It should be in the UI, agree with that.